### PR TITLE
Migrating zendframework to laminas

### PR DESCRIPTION
**Pull request by orklah**
As discussed on #542, using zendframework is now discouraged in favor of laminas.

I kept zendframework in the conflict section but I'm not an expert on this so I may be wrong

### CODE REVIEW

**Comment by Ocramius** [comment]
This still requires changing all `Zend\\` references to `Laminas\\` references (should be a find-replace task)

**Comment by orklah** [comment]
I replaced all usages and mentions in code and docs. There is just one left in the changelog on purpose.

**Comment by Ocramius** [comment]
LGTM! Well done, thanks! :+1:

File: examples/remote-proxy.php [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
+use Laminas\XmlRpc\Client;
 
 require_once __DIR__ . '/../vendor/autoload.php';
 
-if (! class_exists('Zend\XmlRpc\Client')) {
-    echo "This example needs Zend\\XmlRpc\\Client to run. \n In order to install it, "
+if (! class_exists('Laminas\XmlRpc\Client')) {
+    echo "This example needs Laminas\\XmlRpc\\Client to run. \n In order to install it, "
         . "please run following:\n\n"
         . "\$ php composer.phar require zendframework/zend-xmlrpc:2.*\n\n";
```
Comment (victorlap) [inline examples/remote-proxy.php]: Should this line be changed as well?

Comment (orklah) [reply examples/remote-proxy.php]: The Class can't exists if it's not imported and we can't import it without having it in composer so I believe it should.

However, the require command still mentions zendframework. This should be changed too

Comment (Ocramius) [reply examples/remote-proxy.php]: Handled in #550

**Pull request merged**